components/card/card-footer-link.js

```diff
@@ -151,26 +127,26 @@ class CardFooterLink extends RtlMixin(LitElement) {
 				target="${ifDefined(this.target)}"
 				type="${ifDefined(this.type)}">
 				<span class="d2l-offscreen">${this.text}</span>
+				<d2l-count-badge-icon
+					aria-hidden="true"
+					icon="${this.icon}"
+					max-digits="${ifDefined(this.secondaryCountMaxDigits ? this.secondaryCountMaxDigits : undefined)}"
+					number="${noNumber ? 0 : this.secondaryCount}" 
```

Comment:
Passing the number 0 will render a zero on the badge. If consumers want no counts to show up, the secondary-count should not be set.